src/components/Bookstate.js

```diff
@@ -0,0 +1,41 @@
+export default function Bookstate( // eslint-disable-next-line react/prop-types
+  {
+  // eslint-disable-next-line react/prop-types
+    itemid, title, category, removeBook,
+  },
+) {
+  const handleRemove = () => {
+    removeBook(itemid);
+  };
+  return (
+    <div className="book-container">
+      <div className="book-detail-container">
+        <h5>{category}</h5>
+        <h2>{title}</h2>
+        <ul>
+          <button
+            type="button"
+            className="detail-btn"
+          >
+            Comments
+          </button>
+          <span />
+          <button
+            className="detail-btn"
+            type="button"
+            onClick={handleRemove}
+          >
+            Delete
+          </button>
+          <span />
+          <button
+            className="detail-btn"
+            type="button"
+          >
+            Edit
+          </button>
+        </ul>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] [optional] @addisalemseifu Great work on your book state component, however, I would suggest you change it to something like the code below, this will ensure you avoid having linter errors and have a better code for rendering the books
```suggestion
import React from 'react';

function Book(props) {
  // Destructure the book data and the removeBook method from the props
  const { id, title, author, removeBook } = props;

  // A handler function to call the removeBook method with the book id
  const handleRemove = () => {
    removeBook(id);
  };

  return (
    <div className="book">
      {/* Render the title and the author of the book */}
      <h3>{title}</h3>
      <p>{author}</p>
      {/* Render a button for removing books */}
      <button onClick={handleRemove}>Remove</button>
    </div>
  );
}

export default Book;
```